com.playeveryware.eos/Runtime/Core/EOSManager.cs

```diff
@@ -611,6 +595,8 @@ private void Init(IEOSCoroutineOwner coroutineOwner, string configFileName)
 
                 if (!string.IsNullOrWhiteSpace(epicArgs.epicSandboxID))
                 {
+                    // TODO: Move this to set the deployment on the platform config
+                    //       unless that is done, this will no longer function.
```

Comment:
Yeah this PR can't be approved until this is addressed.

Reply:
Some investigation will be needed to determine what the best course of action is.

Reply:
The identical behavior was implemented, but using `PlatformConfig` and `ProductConfig` values to verify that the sandbox id specified is a defined and valid sandbox to use.